pom.xml

```diff
@@ -81,7 +81,7 @@
         <dependency>
             <groupId>io.opentracing</groupId>
             <artifactId>opentracing-impl</artifactId>
-            <version>0.20.2</version>
+            <version>0.20.3-SNAPSHOT</version>
```

Comment:
will be non-snapshot before being merged, after https://github.com/opentracing/opentracing-java/pull/71